### PR TITLE
added files to allow compiling on ubuntu 64bit

### PR DESCRIPTION
**Pull request by KuriKai**
had to change
 if gcc $CommonCompilerFlags $CommonLinkerFlags../../src/war1.c ../../deps/include/glad/glad.c -o war1; then 
to 
 if gcc $CommonCompilerFlags ../../src/war1.c ../../deps/include/glad/glad.c -o war1 $CommonLinkerFlags; then

for it to compile

### CODE REVIEW

**Comment by acoto87** [comment]
Thanks for the pull request! and for testing it your system. I just have a minor questions (posted as separate comments)

The change to put the **$CommonLinkerFlag** to the end of the line is one I already needed to do for windows build files, so it's fine.

File: build-gcc-linux64.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Lib path (relative to output folder)
+LibPath="../../deps/lib/linux64"
+
+# Compiler flags
+# ProfilerFlags="-pg"
+# OptimizeFlags="-O2"
+# AssemblyFlags="-g -Wa,-ahl"
+# DebugFlags="-g -D __DEBUG__"
+CommonCompilerFlags="-std=c99 -Wall -x c $ProfilerFlags $OptimizeFlags $AssemblyFlags $DebugFlags -I $IncludePath"
+CommonLinkerFlags="-L $LibPath -lglfw -lpthread -lm -ldl -lXcursor -g"
```
Comment (acoto87) [inline build-gcc-linux64.sh]: Is xcursor library needed here? it's a dependency of GLFW on Ubuntu?

File: deps/lib/linux64/libglfw.so [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/dave/src/war1/deps/lib/linux64/libglfw.so.3.3
```
Comment (acoto87) [inline deps/lib/linux64/libglfw.so]: Why is this absolute path here?

**Comment by KuriKai** [comment]
Whoops. I'll fix that. And I'll test the xcusor thing too

**Comment by KuriKai** [comment]
@acoto87 Fixed a few things

Hopefull this is good(first time trying to do pull requests)

**Comment by acoto87** [comment]
PR looks good. 

Although, I've been reading about the **-no-pie** linker option, and it seems that what I want and what is usual nowdays is the opposite. I read the thread you mention in a comment, and it seems that that issue is for a particular file browser on gnome and doesn't affect all distribution or other file browsers and that you can always start the program through the CLI.

However, as of now, I can't test the game on any distribution but Raspbian, I'll leave it for now, and then come back to analyze it further.

Gonna merge!

**Comment by acoto87** [comment]
Thanks for your contribution! :tada:

**Pull request merged**